C++/C.sublime-syntax

```diff
@@ -193,6 +193,18 @@ contexts:
         - include: else-pop
     - match: (?=\S)
       fail: try-function
+  expression.try-call:
+    - match: ({{std_funcs}})\s*$
+      captures:
+        1: variable.function.c support.function.std.c
+    - match: ({{identifier}})\s*$
+      captures:
+        1: variable.function.c
+    - match: \(
+      scope: punctuation.section.group.begin.c
+      set: enclosure.parenthesis
+    - match: (?=\S)
+      scope: invalid.illegal.bad
```

Comment:
A lookahead doesn't consume anything and thus doesn't receive any scope.

You most likely want to `fail: try-call`?

Reply:
oops this is an artifact from debugging yes you are correct